exec-service-wasmer/src/wasmer_instance.rs

```diff
@@ -52,6 +52,43 @@ impl WasmerInstance {
         }))
     }
 
+    pub(crate) unsafe fn try_new_instance_from_cache(
```

Comment:
Is this method safe in general?
If so, maybe we should just wrap Module::deserialize in an unsafe block.

Reply:
done